### PR TITLE
feat: support more zil eco projects

### PR DESCRIPTION
**Pull request by renlulu**

### CODE REVIEW

File: projects/xanpool/project.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (AshIgnYeo) [inline projects/xanpool/project.md:1]: Xanpool seems to have some formatting issues. frontmatter doesn't appear properly

**Review by AshIgnYeo — CHANGES_REQUESTED** [review]

File: projects/valkyrie/project.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Valkyrie
+description: A specialized alternative asset management firm at the intersection of traditional finance and the emerging cryptocurrency sector. 
+categories: false
```
Comment (AmritKumar) [inline projects/valkyrie/project.md:4]:
```suggestion
categories: fund
```

File: projects/valkyrie/project.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+status: False
+twitter: False
```
Comment (AmritKumar) [inline projects/valkyrie/project.md:6]:
```suggestion
twitter: https://twitter.com/ValkyrieFunds
```

File: projects/valkyrie/project.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+website: False
```
Comment (AmritKumar) [inline projects/valkyrie/project.md:7]:
```suggestion
website: https://valkyrieinvest.com/
```

**Review by AshIgnYeo — APPROVED** [review]

**Pull request merged**